pressbooks/pressbooks-multi-institution
# feat: limit multiselect selection

This PR updates the https://github.com/greatislander/pressbooks-multiselect component to limit the number of selected users. Addresses #116 

**How to test**

In the institution form, try to select institutional managers. You should be able to only select a maximum of 3 options.
